.github/workflows/build_iso.yml

```
@@ -147,17 +147,34 @@ jobs:
           additional_templates: '/github/workspace/installer/lorax_templates/remove_root_password_prompt.tmpl /github/workspace/installer/lorax_templates/set_default_user.tmpl'
           repos: '/github/workspace/bazzite.repo /etc/yum.repos.d/fedora.repo /etc/yum.repos.d/fedora-updates.repo'
 
-      - name: Move ISOs to Upload Directory
+      - name: Generate a .torrent file
+        id: create-torrent
+        shell: bash
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y mktorrent
+          TORRENT_DIR=${{ github.workspace }}/torrent
+          mkdir ${TORRENT_DIR}
+          cd ${TORRENT_DIR} && mktorrent \
+            --announce="http://torrent.fedoraproject.org:6969" \
```

```suggestion
            --announce="udp://fosstorrents.com:6969/announce" \
            --announce="http://fosstorrents.com:6969/announce" \
```
As the fedora project tracker is not open I suggest using fosstorrents's tracker

---

[I will clean up the commits in a bit]